README.md

```diff
@@ -315,9 +316,9 @@ webpack:
         binary: '/path/to/node'
         node_modules_path: '%kernel.root_dir%/../node_modules'
     output:
-        path: '%kernel.root_dir%/../web/compiled'
-        dump_path: '/bundles'
-        public_path: '/compiled'
+        path: '%kernel.root_dir%/../web/compiled/'
+        dump_path: '%kernel.root_dir%/../web/bundles/'
+        public_path: '/compiled/'
```

Comment:
Why do this with the dump_path and not with the public_path?
Doesn't make much sense to me, and feels very inconsistent.